src/main/java/pl/psnc/dei/ui/pages/ImportsHistory.java

```diff
@@ -13,11 +13,11 @@
 @Route(value = "history", layout = MainView.class)
```

Comment:
There is an exception when you try to navigade directly to history (https://localhost:8443/history):

`org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'pl.psnc.dei.ui.pages.ImportPage' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
`